applications/wg-easy/container/Containerfile

```diff
@@ -63,7 +63,14 @@ RUN curl https://raw.githubusercontent.com/helm/helm/main/scripts/get-helm-3 | b
     | cut -d : -f 2,3 \
     | tr -d \") -o replicated.tar.gz \
     && tar xf replicated.tar.gz replicated && rm replicated.tar.gz \
-    && mv replicated /usr/local/bin/replicated
+    && mv replicated /usr/local/bin/replicated \
+
+    # Install yq
+    && BINARY=yq_linux_amd64 \
+    && VERSION=v4.45.1 \
+    && curl -Ls https://github.com/mikefarah/yq/releases/download/${VERSION}/${BINARY}.tar.gz -O \
```

Comment:
from https://github.com/replicatedhq/platform-examples/pull/59, Chris asks "Why did you move yq out of the package manager and into manual install? Are we using something that's so new it's not in packages yet?"